src/CodeReviews.coffee

```diff
@@ -676,6 +677,7 @@ class CodeReviews
               newStatus = false
               message = "Hey @#{cr.user.name}, looks like *#{cr.slug}* was closed on GitHub." +
               " Say `ignore #{cr.slug}` to remove it from the queue."
```

Comment:
Since every room can have it's own code review queue... and this message used to appear in-channel to identify which queue/channel, it seems like we might want to append the channel name to this message in some manner